packages/gatsby-source-prismic-graphql/README.md

```diff
@@ -64,6 +64,7 @@ yarn add gatsby-source-prismic-graphql
       component: require.resolve('./src/templates/article.js'),
       sortBy: 'date_ASC', // optional, default: meta_lastPublicationDate_ASC; useful for pagination
     }],
+    extraPageFields: 'article_type', // optional, extends pages query to pass extra fields
```

Comment:
Doesn't this only add a single field, rather than multiple field**s**?

Reply:
```
  extraPageFields: `
      article_type
      article_something_else
  `
```

So you could pass the string like the above to add different fields, if you make an error in this the query will fail so we'll get feedback on mistakes in the string... But I'm thinking we can either make it a single field to keep it robust or an array of string and format them in / around `getDocumentsQuery`. What do you think?

Reply:
I'd probably make extraPageFields accept a string OR an array of strings, and format them before inserting them into the query.